src/containers/Permission.jsx

```diff
@@ -1,32 +1,64 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
 
-import { APPS_DOCTYPE } from 'doctypes'
+import { APPS_DOCTYPE, KONNECTORS_DOCTYPE } from 'doctypes'
 import { withRouter } from 'react-router-dom'
 import Typography from 'cozy-ui/transpiled/react/Typography'
-import { Q, useQuery, isQueryLoading, hasQueryBeenLoaded } from 'cozy-client'
+import CozyClient, {
+  Q,
+  useQuery,
+  isQueryLoading,
+  hasQueryBeenLoaded
+} from 'cozy-client'
 import { useI18n } from 'cozy-ui/transpiled/react'
 import Page from 'components/Page'
 import Spinner from 'cozy-ui/transpiled/react/Spinner'
-import List from 'cozy-ui/transpiled/react/MuiCozyTheme/List'
-import ListItemText from 'cozy-ui/transpiled/react/ListItemText'
+import { displayPermissions } from './helpers/permissionsHelper'
 
 const Permission = ({ match }) => {
   const { t } = useI18n()
   const appName = match.params.app
   const permissionName = match.params.permission
+  const THIRTY_SECONDS = 30 * 1000
 
-  const queryResult = useQuery(
+  const [verbs, setVerbs] = useState([])
+
+  const queryResultApps = useQuery(
     Q(APPS_DOCTYPE).getById('io.cozy.apps/' + appName),
     {
-      as: 'io.cozy.apps/' + appName
+      as: 'io.cozy.apps/' + appName,
+      fetchPolicy: CozyClient.fetchPolicies.olderThan(THIRTY_SECONDS)
     }
   )
 
+  const queryResultKonnectors = useQuery(
```

Comment:
There are 2 queries run each time, one for apps, one for konnectors. This is not ideal because 1 query is always  useless. We should fix this in the future, could add a FIXME or TODO in comment?